website/_docs/tutorial/kylin_function.cn.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: docs-cn
+title:  "SQL 函数"
+categories: tutorial
+permalink: /cn/docs/tutorial/kylin_function.html
+---
+
+## 函数
+
+[COUNT](#COUNT)
+　[COUNT(COLUMN)](#COUNT_COLUMN)
+　[COUNT(*)](#COUNT_)
+[COUNT_DISTINCT](#COUNT_DISTINCT)
+[EXTENDED_COLUMN](#EXTENDED_COLUMN)
+[MAX](#MAX)
+[MIN](#MIN)
+[PERCENTILE](#PERCENTILE)
+[SUM](#SUM)
+[TOP_N](#TOP_N)
+
+[WINDOW](#WINDOW)
+　[ROW_NUMBER](#ROW_NUMBER)
+　[AVG](#AVG)
+　[RANK](#RANK)
+　[DENSE_RANK](#DENSE_RANK)
+　[FIRST_VALUE](#FIRST_VALUE)
+　[LAST_VALUE](#LAST_VALUE)
+　[LAG](#LAG)
+　[LEAD](#LEAD)
+　[NTILE](#NTILE)
+　[CASE WHEN](#CASEWHEN)
+　[CAST](#CAST)
+
+[SUSTRING](#SUBSTRING)
+[COALESCE](#COALESCE)
+
+
+## COUNT {#COUNT}
+用于返回与指定条件匹配的行数。
+
+### COUNT(COLUMN) {#COUNT_COLUMN}
+
+例子：
+{% highlight Groff markup %}
+SELECT COUNT(seller_id) FROM kylin_sales;
+{% endhighlight %}
+
+### COUNT(*) {#COUNT_}
+
+例子：
+{% highlight Groff markup %}
+SELECT COUNT(*) FROM kylin_sales;
+{% endhighlight %}
+
+
+## COUNT_DISTINCT {#COUNT_DISTINCT}
+
+例子：
+{% highlight Groff markup %}
+SELECT COUNT(DISTINCT seller_id) AS DIST_SELLER FROM kylin_sales;
+{% endhighlight %}
+
+## EXTENDED_COLUMN {#EXTENDED_COLUMN}
```

Comment:
This is not a function, we can remove it.

Reply:
ok